packages/fscomponents/src/components/ZoomCarousel/ZoomCarousel.web.tsx

```diff
@@ -303,18 +306,23 @@ export class ZoomCarousel extends Component<ZoomCarouselProps, ZoomCarouselState
 
   render(): JSX.Element {
     const { peekSize = 0, gapSize = 0 } = this.props;
+    const height =
+      this.props.fillContainer ?
+      (this.props.showThumbnails ? S.fullHeightWithThumbnails : S.fullHeight)
+       : null;
 
+    // line 316 and 324 needs to be at 100% height
     return (
       <View
-        style={this.props.fillContainer ? S.fullHeight : null}
+        style={S.fullHeight}
         onLayout={this.handleLayoutChange}
       >
         <View
-          style={this.props.fillContainer ? S.fullHeight : null}
+          style={height}
         >
           <div
             id={`zoom-carousel-${this.id}`}
-            style={this.props.fillContainer ? {height: '100%'} : undefined}
+            style={this.props.fillContainer ? S.fullHeight : undefined}
```

Comment:
I tried running StoryBook and got a typing error on your changes here. I think the problem is that RN web doesn't know how to handle a RN StyleSheet element inside of a HTML element. You can either revert this specific change to fix the error, or you can change `<div>` to the following:

```
<View
  nativeID={`zoom-carousel-${this.id}`}
  style={this.props.fillContainer ? S.fullHeight : undefined}
>
```

Note that `nativeID` will become `id` when rendered as HTML.

Reply:
Reverted back